core/server/models/session.js

```diff
@@ -0,0 +1,49 @@
+const ghostBookshelf = require('./base');
+
+const Session = ghostBookshelf.Model.extend({
+    tableName: 'sessions',
+
+    parse: function parse(attrs) {
+        attrs.session_data = JSON.parse(attrs.session_data);
+        return attrs;
+    },
+
+    format: function format(attrs) {
+        if (attrs.session_data) {
```

Comment:
`session_data` is not optional, right?

Reply:
Nope but format _can_ get called without a complete model, I get errors when fetching sometimes if this is missing.

Reply:
Have you debugged?

Reply:
Yeah, e.g. `findOne` will call format on the properties passed to it (which don't always include session_data)

Reply:
Okay 👍 Wanne leave a comment?

Reply:
Sure :smiley_cat: